Remove one spelling error in the binary.
--- a/src/loader/loader_appleii.c	2022-04-30 09:36:03.231952692 +0200
+++ b/src/loader/loader_appleii.c	2022-04-30 09:36:21.339740702 +0200
@@ -782,7 +782,7 @@
 		}
 		if (gameid==-1 || gameid==newgameid) gameid=newgameid;
 		else {
-			fprintf(stderr,"Game detection ambigous\n");
+			fprintf(stderr,"Game detection ambiguous\n");
 			return -1;
 		}
 		if (disknum>=0 && disknum<MAXDISKS)
